apps/docs/src/app/core/component-docs/object-marker/examples/object-marker-text-example.component.html

```diff
@@ -1,4 +1,10 @@
-<span fd-object-marker aria-label="Object Maker with Text" label="Draft"></span>
-<span fd-object-marker aria-label="Object Maker with Text" label="Request"></span>
-<a fd-object-marker clickable="true" aria-label="Object Maker with only Text Clickable" label="Locked"></a>
-<a fd-object-marker clickable="true" aria-label="Object Maker with only Text Clickable" label="Edit"></a>
+<span fd-object-marker aria-label="Object Marker with Text" label="Draft"></span>
+<span fd-object-marker aria-label="Object Marker with Text" label="Request"></span>
+<a
+    fd-object-marker
+    clickable="true"
+    aria-label="Object Marker with only Text Clickable"
+    label="Locked"
+    tabindex="0"
```

Comment:
Isn't this weird that you need to explicitly add tabindex? It should not be like that, anchor tag is tabbable by default.